install_config/aggregate_logging.adoc

```diff
@@ -1011,6 +1012,121 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
   done
 ----
 
+[[aggregated-logging-hostPath-Elasticsearch]]
+===== Configuring hostPath storage for Elasticsearch =====
+
+You can provision {product-title} clusters using hostPath storage for Elasticsearch.
+
+To use a local disk volume on each node host as storage for an Elasticsearch replica:
+
+. Create a local mount point on each infrastructure node for the local Elasticsearch storage:
++
+----
+mkdir /usr/local/es-storage
```

Comment:
This command needs a $ in front of it:

$ mkdir /usr/local/es-storage

Reply:
Added. Thanks.